packages/components/inputs/time-input/src/time-input-body.styles.js

```diff
@@ -1,9 +1,8 @@
 import { css } from '@emotion/core';
 import styled from '@emotion/styled';
-import {
-  customProperties as vars,
-  designTokens,
-} from '@commercetools-uikit/design-system';
+import { customProperties as vars } from '@commercetools-uikit/design-system';
+import AccessibleButton from '@commercetools-uikit/accessible-button';
+
```

Comment:
Main changes in these styles:
Borders and background were moved to the container, so they that their state (disabled, hasError, focused, etc) is easier to control, only in one place.